### PR TITLE
Add missing doc

### LINKED ISSUE #369: FR: `Format Yaml Array` rule doesn't have a setting to disable formatting tags
`Format Yaml Array` rules has a setting `Format yaml aliases section` but doesn't have corresponding `Format yaml tags section`

### PR DESCRIPTION
**Pull request by mnaoumov**
Fixes #369 

Add missing changes from #370

### CODE REVIEW

File: docs/rules.md [hunk 1]
```diff
@@ -211,6 +211,8 @@ Options:
 	- `single-line space delimited`: ```tags: [tag1 tag2]```
 	- `single string space delimited`: ```tags: tag1 tag2```
 	- `single string comma delimited`: ```tags: tag1, tag2```
+- Format yaml tags section: Turns on formatting for the yaml tags section.
```
Comment (platers) [inline docs/rules.md:214]: This file is auto generated from rules by docs.ts, you probably want to make this change somewhere else.

Comment (mnaoumov) [reply docs/rules.md:214]: Yes, this file is indeed autogenerated. If it is not meant to be committed, then I suggest this file to be added to the `.gitignore`

Comment (pjkaufman) [reply docs/rules.md:214]: @mnaoumov , the changes listed here were created by running `doc.js`  via the npm commands, right? If so, then we are good to have this merged.

I just want to clear up any questions around how this change was created.

Comment (platers) [reply docs/rules.md:214]: Oh sorry, confused myself. Yep, I think we should keep it this way, I don't see another way to host the docs.

**Pull request merged**